crmsh/sbd.py

```diff
@@ -241,6 +308,30 @@ def _enable_sbd_service(self):
             # in init process
             bootstrap.invoke("systemctl enable sbd.service")
 
+    def _adjust_systemd(self):
+        """
+        Adjust start timeout for sbd when set SBD_DELAY_START
+        """
+        if not self.is_delay_start():
+            return
+
+        # TimeoutStartUSec default is 1min 30s, need to parse as seconds
+        cmd = "systemctl show -p TimeoutStartUSec sbd --value"
+        out = utils.get_stdout_or_raise_error(cmd)
+        res_seconds = re.search("(\d+)s", out)
+        default_start_timeout = int(res_seconds.group(1)) if res_seconds else 0
+        res_min = re.search("(\d+)min", out)
+        default_start_timeout += 60 * int(res_min.group(1)) if res_min else 0
+        sbd_watchdog_timeout = SBDManager.get_sbd_watchdog_timeout()
+        if default_start_timeout >= (sbd_watchdog_timeout * 2):
```

Comment:
Technically we should check it against `msgwait` for the case with disk.

Reply:
Changed to compare `if default_start_timeout >= self.get_sbd_start_timeout_threshold():`
That will make code more reusable

Thanks!